src/Sulu/Component/Content/Document/Behavior/BlameBehavior.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Content\Document\Behavior;
+
+use Sulu\Component\Security\Authentication\UserInterface;
+
+/**
+ * Adds the user who created and lastly changed the document.
+ */
+interface BlameBehavior
+{
+    /**
+     * @return UserInterface
+     */
+    public function getCreator();
+
+    /**
+     * @return UserInterface
+     */
+    public function getChanger();
```

Comment:
Worth a thought, but I only moved this interface from the [document manager](https://github.com/sulu-io/sulu-document-manager), and if we adapt this method name now it would be a BC break.